### PR TITLE
【Prompt】fix prompt compute loss without criterion

### PR DESCRIPTION
**Pull request by lugimzzz**
### PR types
Bug fixes 

### PR changes
APIs

### Description
修复prompt没有定义criterion的情况下报错

### CODE REVIEW

File: paddlenlp/prompt/prompt_trainer.py [hunk 1]
```diff
@@ -248,7 +248,7 @@ def compute_loss(self, model, inputs, return_outputs=False):
             if self.args.use_rgl:
                 loss += self._compute_rgl_loss(hidden_states, labels)
         else:
-            loss, logits, _ = model(**input_dict)
+            loss, logits = model(**input_dict)
```
Comment (lugimzzz) [inline paddlenlp/prompt/prompt_trainer.py:251]: 因为走的是这一行的return ，所以只有两个参数https://github.com/PaddlePaddle/PaddleNLP/blob/develop/paddlenlp/prompt/prompt_model.py#L125

**Review by sijunhe — APPROVED** [review]
lgtm
